theme-base/README.md

```
@@ -0,0 +1,32 @@
+
+Run the build command from the project root:
+```bash
+npm run build
+```
+
+## Maintenance Benefits
+
+- **Single source of truth**: Token colors defined once
+- **Consistent updates**: Changes to base files apply to all variants
+- **Reduced file size**: ~95% reduction in duplicated code
+- **Easy to add variants**: Just add a new entry to `variants.json`
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)